sc_rf_service/sc_rf_service.py

```diff
@@ -204,12 +272,33 @@ class CavityPVGroup(PVGroup):
                                                        "PARKED"))
     cudDesc: PvpropertyChar = pvproperty(value="Name", name="CUDDESC",
                                          dtype=ChannelType.CHAR)
-
-    def __init__(self, prefix, length):
-
+    ssa_latch: PvpropertyEnum = pvproperty(value=1, name="SSA_LTCH",
+                                           dtype=ChannelType.ENUM,
+                                           enum_strings=("OK", "Fault"))
```

Comment:
What's the difference between this ssa_latch fault and the one at the beginning of `class CavFaultPVGroup(PVGroup):`? I'm wondering if this is a duplicate of line 156:

latch: PvpropertyEnum = pvproperty(value=1, name="SSA_LTCH",
                                       dtype=ChannelType.ENUM,
                                       enum_strings=("OK", "Fault"))

Reply:
Ah good catch! I think that happened when I tried to resolve some merge conflicts...